apps/ca.c

```diff
@@ -246,7 +252,8 @@ int ca_main(int argc, char **argv)
     const char *serialfile = NULL, *subj = NULL;
     char *prog, *startdate = NULL, *enddate = NULL;
     char *dbfile = NULL, *f, *randfile = NULL;
-    char buf[3][BSIZE];
+    char new_cert[CERT_MAX + 1];
```

Comment:
And in the context of above commentary standard PATH_MAX accounts for trailing zero char...